src/base_classes/sports.py

```
@@ -172,7 +172,7 @@ def _get_logo_directory_fallbacks(self, configured_dir: Path) -> List[Path]:
 
         try:
             fallbacks.append(Path.home() / ".ledmatrix" / "logos" / self.sport_key)
-        except Exception:
+        except Exception:  # nosec B110 - Path.home() raises RuntimeError for service users; fallback list continues
             pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)